### PR TITLE
fix: Wrong color in widget see new message button

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

Fixes https://github.com/chatwoot/chatwoot/issues/5977

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## How Has This Been Tested?

**Screenshot**

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/widget/components/UnreadMessageList.vue [hunk 1]
```diff
@@ -27,7 +27,12 @@
         class="button clear-button"
         @click="openConversationView"
       >
-        <span class="flex items-center">
+        <span
+          class="flex items-center"
+          :style="{
+            color: widgetColor,
```
Comment (pranavrajs) [inline app/javascript/widget/components/UnreadMessageList.vue:33]: One problem I can see is that if the user selected white, the button would become invisible.

Comment (iamsivin) [reply app/javascript/widget/components/UnreadMessageList.vue:33]: @pranavrajs Is is fine? 
image

https://github.com/chatwoot/chatwoot/pull/6033/commits/3e06cb6d928fdf5b1d2a81fabbb50efa2792ff60

**Comment by iamsivin** [comment]
@pranavrajs @nithindavid Made some changes in the SDK file and will change the colour dynamically in the lighter background colours. https://github.com/chatwoot/chatwoot/pull/6033/commits/abde10607f0e5308ac01e794b2e250b9af81419b

https://user-images.githubusercontent.com/64252451/206650920-026eb6c2-5760-46d8-9cfc-debfd1ee7fe7.mov

**Comment by iamsivin** [comment]
@pranavrajs, Can you take a look?
1. Updated bubble icon from `img` to `svg`.
2. The text and bubble colour in the widget bubble and the `show new message` button colour will update dynamically as per the brightness of the background. If its lighter, it will change to the default colour `#37546D`(s-700)

@nithindavid, I Updated the PR with the changes you suggested in the call. Can you take a look?

**Preview**

https://user-images.githubusercontent.com/64252451/206752916-4d7a835b-cc24-4d4c-a79f-f8eb905442e9.mov

File: app/javascript/sdk/sdk.js [hunk 2]
```diff
@@ -107,7 +107,11 @@ export const SDK_CSS = `
   width: auto !important;
 }
 
-.woot-widget-bubble.woot-widget--expanded img {
+.woot-widget-bubble.woot-widget--expanded.woot-widget-bubble-color--lighter div{
+  color: 	#37546D;
```
Comment (nithindavid) [inline app/javascript/sdk/sdk.js:111]: Please move this as a variable since we are using this in 3 places

**Review by nithindavid — CHANGES_REQUESTED** [review]

File: app/javascript/sdk/sdk.js (hunk 2 above)
Comment (iamsivin) [reply app/javascript/sdk/sdk.js]: Done

**Review by nithindavid — APPROVED** [review]
Tested in local and working as expected.

**Pull request merged**